.golangci.yml

```diff
@@ -72,6 +72,34 @@ linters-settings:
       # Controller Runtime
       - pkg: sigs.k8s.io/controller-runtime
         alias: ctrl
+      # CAPI
+      - pkg: sigs.k8s.io/cluster-api/api/v1alpha3
```

Comment:
I wonder, if source file imports only single package named `v1alpha3`, will the linter still require an alias?

Reply:
It will, because we have [`no-unaliased: true`](https://golangci-lint.run/usage/linters/#importas) set in [.golangci.yml](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/.golangci.yml#L59).

Reply:
All good if that's deliberate. I think I like it this way, similar to `,` ending the last entry in the lists. It won't make you change unrelated stuff when you add new import. Feel free to mark as resolved.